tasks/add-fbc-contribution/tests/test-add-fbc-contribution-pre-ga.yaml

```diff
@@ -82,16 +82,18 @@ spec:
           - name: check-result
             image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
             script: |
-              #!/usr/bin/env sh
+              #!/usr/bin/env bash
               #
               set -eux
 
               internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers | \
                 tac | tail -1)"
 
-              requestParams=$(kubectl get internalrequest ${internalRequest} -o jsonpath="{.spec.params}")
+              internalRequest=$(echo "${internalRequest}" | xargs)
```

Comment:
why this change?

Reply:
I added quotes to `${internalRequest}` to satisfy Checkton :`"${internalRequest}"`
Once I added the quotes, the test started failing, and from logs, I found extra spaces after the `internal request `

```
[check-result : check-result] ++ tac
[check-result : check-result] ++ tail -1
[check-result : check-result] + internalRequest='iib-hwgz2         '
[check-result : check-result] ++ kubectl get internalrequest 'iib-hwgz2         ' -o 'jsonpath={.spec.params}'
[check-result : check-result] Error from server (NotFound): internalrequests.appstudio.redhat.com "iib-hwgz2         " not found
```

So, I used `xargs` to trim extra spaces.